### DISCUSSION TITLE
Sharing AI papers datasets

### DISCUSSION THREAD

**Discussion by george-gca** [asreview/asreview#1488] [Q&A]
I was finally able to create data to be used in ASReview for some major conferences in AI, like [CVPR](https://cvpr2023.thecvf.com/), and I believe it would be useful for other people. Is there a recommended way to make this data available?

**Comment by Rensvandeschoot** [comment]
Sharing data is always a good idea! :-) 

For reproducibility and citation, it's recommended to publish your dataset with a Digital Object Identifier (DOI) which can be done through platforms like the Open Science Framework (OSF). This gives your data a unique, persistent identifier, enhancing its discoverability.  Also, ensure your data is covered under an appropriate license. A permissive license, such as Creative Commons Attribution (CC BY), would suffice, as it requires others to credit you for your work even if they adapt or build upon it.

Should your dataset include labels, it could be very useful for our benchmark platform at https://github.com/asreview/synergy-dataset. This is a platform for comparing and benchmarking automated systematic review tools and machine learning models.

**Reply by george-gca** [reply]
It is not labeled, but it has the basic information like title, abstract, and link to the publication on the conference. Is there some kind of repository that I could be able to upload this to? Or should I create a repo here in GitHub and somehow share the link?
